lib/data/bank-holidays.json

```diff
@@ -362,6 +362,56 @@
                 "bunting": true
               }
             ],
+            "2019": [
+                {
+                    "title": "bank_holidays.new_year",
+                    "date": "01/01/2019",
+                    "notes": "",
+                    "bunting" :true
```

Comment:
Weird whitespace here.

Reply:
Fixed in https://github.com/alphagov/calendars/pull/152/commits/5068a290d0df375bbe6bea87b499326134dabdea

Reply:
Still seeing varieties of:
```
"bunting" :true
"bunting":true
"bunting": true
```

Should all match the last one?

Reply:
😓  fixed -https://github.com/alphagov/calendars/pull/152/commits/d435987e3c39f9b0792181397c4f9086483833c6